channels/uk.m3u

```diff
@@ -377,6 +413,12 @@ https://ooyalahd2-f.akamaihd.net/i/mtach7_delivery@348438/master.m3u8
 https://ooyalahd2-f.akamaihd.net/i/mtaengaudio_delivery@138280/master.m3u8
 #EXTINF:-1 tvg-id="nTVUSEastern.uk" tvg-name="n TV (US Eastern)" tvg-country="UK" tvg-language="" tvg-logo="https://i.imgur.com/l75bDTx.png" group-title="",n TV (US Eastern)
 https://a.jsrdn.com/broadcast/22680_3BR3zocwi9/-0500/c.m3u8
+#EXTINF:-1 tvg-id="Nick.uk" tvg-name="Nick" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/6riwdSg.png" group-title="Kids",Nick
+http://stream.tvtap.live:8081/live/nick.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="NickJunior.uk" tvg-name="Nick Jr" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/G3e8rWz.jpg" group-title="Kids",Nick Jr
```

Comment:
Originates from the US. (pretty sure we have it too)

Reply:
https://en.wikipedia.org/wiki/Nick_Jr._(British_and_Irish_TV_channel)

I'm starting to think there's always a British-specific version of the US channels.

Reply:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.